tests/safeds/ml/classical/regression/test_regressor.py

```diff
@@ -243,7 +295,7 @@ def test_should_raise_if_validation_fails(
         expected: list[str | int],
         error: type[Exception],
     ) -> None:
-        actual_column = Column("actual", pd.Series(actual))
-        expected_column = Column("expected", pd.Series(expected))
+        actual_column: Column = Column("actual", pd.Series(actual))
+        expected_column: Column = Column("expected", pd.Series(expected))
```

Comment:
Generally, type hints are not required for local variables. In this case in particular, their type is very obvious since we call the `Column` constructor.

Reply:
MyPy raised two errors without these type hints:
tests/safeds/ml/classical/regression/test_regressor.py:292: error: Need type annotation for "actual_column"  [var-annotated]
tests/safeds/ml/classical/regression/test_regressor.py:293: error: Need type annotation for "expected_column"  [var-annotated]

Reply:
My bad, it's needed because `Column` is generic.